best-practices/javascript.md

```diff
@@ -1,18 +1,12 @@
-# General Javascript Best Practices
+# General JavaScript Best Practices
 
 ## Purpose of this document
 
 - Make everyone aware of and discourage general antipatterns.
 - Make everyone aware of general patterns.
-- Provide a point of reference for what good code looks like at Hx.
+- Provide a point of reference for what good code looks like at Holiday Extras.
 - Get people writing code in a similar way.
 
-Please also see the respective serverside and clientside best practice docs.
-
-## ECMAScript version support
-
-We use _some_ ECMAScript 6 at HX. This is fulfilled by Babel and Node v4 where applicable. We should use these newer es6 features where it makes sense and can improve readability. Please have a look at our technical resources [section on ES6](../technical-resources.md#es6) to familiarise yourself with some of the new features available to us. Please see [below](#what-antipatterns-could-i-run-into-with-my-newfound-es6-power) for advice when using some ES6 features. If we come across any new anti-patterns that arise from using ES6 features we should add them to this document.
```

Comment:
Do we need to mention this at this stage? ECMAScript seems so common now that I don't know if it makes us look "behind the times" if we talk about it like this, what do you think? Happy to be disagreed with.

Reply:
Agreed. If anything, in the rare places where we write vanilla JS and not ES we need to specifically look it up ahah